### ISSUE TITLE
Literature about Impact and Ethics of Social media

### ISSUE THREAD

**Issue by maryi** [maryi/CSCW-workshop4#1]
Please start posting any link/paper you thing is interesting to consider in our workshop.

**Comment by maryi** [comment]
Hi everyone, 
I found something really interesting about Facebook experiments..
Blog with general info: http://www.scilogs.com/from_the_lab_bench/the-facebook-emotion-study-in-a-broader-context/
One of the papers: http://www.pnas.org/content/111/24/8788.full
Video: https://www.youtube.com/watch?v=QxVZYiJKl1Y

What do you think??

**Comment by maryi** [comment]
We could use this video as an introduction for our workshop:
https://www.youtube.com/watch?v=0eUeL3n7fDs

**Comment by areyeslo** [comment]
I agree, this video includes all the facts of how social media is changing the communication. I like this idea to find the impact on daily life mix with the first video including ethics. Thank you for sharing.

I share some papers that I found today. They are about the impact of social media in Software Engineering:
-The Impact of Social Media on Software Engineering Practices and Tools
-Social media for software engineering
-Towards understanding twitter use in software engineering: preliminary findings, ongoing challenges and future questions
-Awareness 2.0: staying aware of projects, developers and tasks using dashboards and feeds

**Comment by maryi** [comment]
Hi @areyeslo,
Can you add links to those papers? so we can easily get access to the resource, thanks :)

**Comment by areyeslo** [comment]
Hi, 

I add the URLs to obtain easy the papers and also Included two interesting papers about the impact of social media on Education.

Focus on Software Engineering:
-The Impact of Social Media on Software Engineering Practices and Tools
http://dl.acm.org.ezproxy.library.uvic.ca/citation.cfm?id=1882435

-Social media for software engineering
http://dl.acm.org.ezproxy.library.uvic.ca/citation.cfm?id=1882370

-Towards understanding twitter use in software engineering: preliminary findings, ongoing challenges and future questions
http://dl.acm.org.ezproxy.library.uvic.ca/citation.cfm?id=1984707

-Awareness 2.0: staying aware of projects, developers and tasks using dashboards and feeds
http://ieeexplore.ieee.org.ezproxy.library.uvic.ca/xpls/abs_all.jsp?arnumber=6062104&tag=1

Focus on Education:
Learning 2.0 - The Impact of Social Media on Learning in Europe
http://citeseerx.ist.psu.edu/viewdoc/download?doi=10.1.1.460.2564&rep=rep1&type=pdf

The effect of Twitter on college student engagement and grades
http://onlinelibrary.wiley.com.ezproxy.library.uvic.ca/doi/10.1111/j.1365-2729.2010.00387.x/pdf

**Comment by srwarnock** [comment]
Hi everyone,

I'm not sure if you have to stick to CSCW in the domain of software engineering or if you can branch out to other practices, but I personally find it interesting how the police use social media to help fight crime and engage the community. Here are a few papers to look at if you're interested (they really get you thinking about ethics concerning social media, and the internet in general):

1. Police embrace social media as crime-fighting tool:
By Heather Kelly, CNN

 http://www.cnn.com/2012/08/30/tech/social-media/fighting-crime-social-media/

2. The Future of Police Image and Ethics:
By Joseph A. Schafer, Associate Professor, Center for the Study of Crime, Southern Illinois                                        University Carbondale, Carbondale, Illinois

 http://www.policechiefmagazine.org/magazine/index.cfm?fuseaction=display_arch&article_id=1209&issue_id=62007

3. Social Media: Law and Order Edition
by James Hyatt

 http://business-ethics.com/2013/05/05/1104-social-media-law-and-order-edition/

**Comment by maryi** [comment]
Hi @srwarnock 
I don't think we need to keep our topic restricted to soft. eng. If the impact of social media is felt in other domains or fields, I think we can talk about it.. :+1: 
PS: Just to be sure, I'll ask Peggy :)

**Comment by maryi** [comment]
Paper: http://dl.acm.org/citation.cfm?id=2002512
Fictious opinions (or user reviews) that have been deliberately written to sound authentic  in recommendation systems.
